app/views/menus/_preview_menus.html.haml

```diff
@@ -0,0 +1,4 @@
+- if object.menus.any?
+  %ul
+    - object.menus.each do |menu|
+      %li= link_to t('menu',day: menu.date.strftime("%m.%d.%Y")), menu_path(menu.id), class: "btn btn-primary btn-sm mt-10"
```

Comment:
space after `t('.menu', day: ....`

you can add `human_date` method in your future `MenuDecorator` and call it instead `menu.date.strftime("%m.%d.%Y")`